test/units/plugins/action/test_synchronize.py

```diff
@@ -99,11 +99,11 @@ class SynchronizeTester(object):
     execute_called = False
 
 
-    def _execute_module(self, module_name, task_vars=None):
+    def _execute_module(self, module_name, module_args=None, task_vars=None):
```

Comment:
_execute_module has a signature of:

```lib/ansible/plugins/action/__init__.py:    def _execute_module(self, module_name=None, module_args=None, tmp=None, task_vars=None, persist_files=False, delete_remote_tmp=True):```

The tests already use an incomplete function signature that is being updated to accept the explicit module_args rather than the implicit args via the task object.

I can update to make the tmp args function local and check against the args passed into this function, but any issue with the tests was preexisting.